.github/ISSUE_TEMPLATE/release.md

```diff
@@ -51,10 +53,12 @@ labels: ["release"]
 
 #### In k6 repository
 
-- [ ] Merge the PR bumping [the k6 Go project's version](https://github.com/grafana/k6/blob/master/lib/consts/consts.go#L11-L12).
+- [ ] Make sure all the patches have been cherry-picked on the release branch.
```

Comment:
What if we're in case that defect doesn't affect the `master` branch but only the release branch? So, does commit continue to exist only in the release branch?

Reply:
Yes, release branch + tag. Do you see issues with it?

Reply:
Yes, it's similar to https://github.com/grafana/k6/pull/3911/files#r1738465282

Reply:
Integrated with a dedicated text.